plugins/Events/lang/en.json

```diff
@@ -28,8 +28,6 @@
         "MaxValueDocumentation": "The maximum value for this event",
         "MinValue": "Minimum Event value",
         "MinValueDocumentation": "The minimum value for this event",
-        "SecondaryDimension": "Secondary dimension is %s.",
```

Comment:
Is there a way in weblate to move all previous translations? Or would they need to be re-translated over time?